rules/downgrade-php73/tests/Rector/FuncCall/SetCookieOptionsArrayToArgumentsRector/Fixture/options_scalars.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+if (false) {
+    setcookie('name', 'value', ['expires' => 3600]);
```

Comment:
These snippets of code can be executed.

Better wrap it in `class SomeClass {}` and `run()` method as other fixture have.
That also prevent missing classes and keeps test fixture format united.